[2024-01-15T16:48:04.069-0300] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cities_novo.saopaulo scheduled__2024-01-15T19:45:43.537388+00:00 [queued]>
[2024-01-15T16:48:04.072-0300] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cities_novo.saopaulo scheduled__2024-01-15T19:45:43.537388+00:00 [queued]>
[2024-01-15T16:48:04.072-0300] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-01-15T16:48:04.099-0300] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): saopaulo> on 2024-01-15 19:45:43.537388+00:00
[2024-01-15T16:48:04.102-0300] {standard_task_runner.py:57} INFO - Started process 85922 to run task
[2024-01-15T16:48:04.105-0300] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'cities_novo', 'saopaulo', 'scheduled__2024-01-15T19:45:43.537388+00:00', '--job-id', '1283', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmppj6oupx7']
[2024-01-15T16:48:04.105-0300] {standard_task_runner.py:85} INFO - Job 1283: Subtask saopaulo
[2024-01-15T16:48:04.134-0300] {task_command.py:416} INFO - Running <TaskInstance: cities_novo.saopaulo scheduled__2024-01-15T19:45:43.537388+00:00 [running]> on host desktop
[2024-01-15T16:48:04.195-0300] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='datasetpibic@gmail.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='cities_novo' AIRFLOW_CTX_TASK_ID='saopaulo' AIRFLOW_CTX_EXECUTION_DATE='2024-01-15T19:45:43.537388+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-15T19:45:43.537388+00:00'
[2024-01-15T16:48:04.359-0300] {logging_mixin.py:154} INFO - true
[2024-01-15T16:48:04.359-0300] {logging_mixin.py:154} INFO - <Response [200]>
[2024-01-15T16:48:05.221-0300] {logging_mixin.py:154} INFO - Called | 1705348085.217658 | sp
[2024-01-15T16:48:05.240-0300] {logging_mixin.py:154} INFO - SAVING FILE sp
[2024-01-15T16:48:05.242-0300] {python.py:194} INFO - Done. Returned value was: None
[2024-01-15T16:48:05.244-0300] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=cities_novo, task_id=saopaulo, execution_date=20240115T194543, start_date=20240115T194804, end_date=20240115T194805
[2024-01-15T16:48:05.318-0300] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-01-15T16:48:05.331-0300] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
